src/index.js

```diff
@@ -58,7 +59,12 @@ class TracePlugin {
       end: this.end.bind(this)
     };
     this.invocationInstance.context.iopipe.measure = this.measure.bind(this);
-    if (typeof this.invocationInstance.context.iopipe.label === 'function') {
+  }
+  postInvoke() {
+    if (
+      typeof this.invocationInstance.context.iopipe.label === 'function' &&
+      this.timeline.data.length > 0
```

Comment:
I would prefer if we used the exposed method on this package which is `getEntries` instead of relying on internals.

Reply:
@coreylight thanks, that's the sort of thing I was looking for!